Doc/library/sqlite3.rst

```diff
@@ -47,85 +47,170 @@ This document includes four main sections:
       PEP written by Marc-André Lemburg.
 
 
+.. We use the following practises for SQL code:
+   - UPPERCASE for keywords
+   - snake_case for schema
+   - single quotes for string literals
+   - singular for table names
+   - if needed, use double quotes for table and column names
+
 .. _sqlite3-tutorial:
 
 Tutorial
 --------
 
-To use the module, start by creating a :class:`Connection` object that
-represents the database.  Here the data will be stored in the
-:file:`example.db` file::
+In this tutorial, you will learn the basics of the :mod:`!sqlite3` API
```

Comment:
I'm not actually keen on "you will learn..." - it seems presumptuous and raises too many questions. What if the user fails to learn, or only learns a part of what you wanted to teach? Is it even possible for a user to learn, in a meaningful way, all these things through just one lesson?

I prefer to say things like "This tutorial is designed to help you learn [whatever]" or "to help you become familiar with", etc. And then:  "you will encounter...", "you will use...", you will build...".

Reply:
I missed this one, but I really should have caught it by now—you stress this multiple times in the tutorial section, and I even quoted 

> Describe what the learner will accomplish (note - not: “you will learn…”).

Reply:
How about this:

```diff
-In this tutorial, you will learn the basics of the :mod:`!sqlite3` API
-by creating a database of Monty Python movies.
+In this tutorial, you will create a database of Monty Python movies,
+using basic :mod:`!sqlite3` APIs.
```